### ISSUE TITLE
Variable Speed Merkel Cooling Tower provides too much free convection cooling

### ISSUE THREAD

**Issue by EnergyArchmage** [NREL/EnergyPlus#4927]
Found that the main calculation routine for merkel vs cooling tower does not handle the case where no cooling is needed at all.  When no cooling is needed or called for, the model is placing the outlet temperature for free convection on the leaving node.  The tower's own output for leaving temperature is correct but does not agree with the system node temperature.  Impact is quite large in that tower continues to cool at the no-fan capacity when no cooling is called for. 

adding this at line 4210 in CondenserLoopTowers.cc appears to solve the problem. 

```
if ( std::abs( MyLoad ) <= SmallLoad ) {
    // tower doesn't need to do anything
        OutletWaterTemp = Node( WaterInletNode ).Temp;
        CTFanPower = 0.0;
        AirFlowRateRatio = 0.0;
        Qactual = 0.0;
        CalcBasinHeaterPower( SimpleTower( TowerNum ).BasinHeaterPowerFTempDiff, SimpleTower( TowerNum ).BasinHeaterSchedulePtr, SimpleTower( TowerNum ).BasinHeaterSetPointTemp, BasinHeaterPower );
        return;
    }
```

**Comment by EnergyArchmage** [comment]
#5101

**Comment by mjwitte** [comment]
Closed via #5178
